youtube_dl/extractor/funimation.py

```diff
@@ -0,0 +1,71 @@
+# coding: utf-8
+from __future__ import unicode_literals
+from .common import InfoExtractor
+from ..compat import compat_HTTPError
+from ..utils import (
+    encode_dict,
+    sanitized_Request,
+    ExtractorError,
+    urlencode_postdata
+)
+import re
+
+class FunimationIE(InfoExtractor):
+    _VALID_URL = r'https?://(?:www\.)?funimation\.com/shows/.*/videos/official/(?P<id>[^?]+)'
```

Comment:
It should not match URLs like http://www.funimation.com/shows//videos/official/breeze.